docs/running-on-kubernetes.md

```diff
@@ -0,0 +1,512 @@
+---
+layout: global
+title: Running Spark on Kubernetes
+---
+* This will become a table of contents (this text will be scraped).
+{:toc}
+
+Spark can run on clusters managed by [Kubernetes](https://kubernetes.io). This feature is currently experimental.
```

Comment:
I think we should call out Kubernetes as a native scheduler backend to differentiate it from other options like helm charts for standalone.